reports/audit-runs/20260825T134536Z-debt-cycle-d9ac42612b/final-summary.md

```
@@ -0,0 +1,40 @@
+
+run_id: `20260825T134536Z-debt-cycle-d9ac42612b`
+
+## Gate: WARN
+
+Paydown is green on `fix/audit-cycle-tech-debt-d9ac42612b`. Not merged (`ALLOW_MERGE=false`).
+
+## surface_score: 2
+
+## Issues
+
+- #9646 [tech-debt][REQ-GOV-012][P2] re-pin
```

markdownlint-cli2 (0.23.2)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "p2"

(MD052, reference-links-images)